tests/services/test_crash_reports.py

```diff
@@ -74,13 +74,21 @@ def test_pull(crash_manager, delete_test_dir):
 
 
 @pytest.mark.parametrize(
-    ('message', 'expected'),
+    ('message', 'return_value'),
     (
         ('sysdiagnose (full) complete', True),
         ('Sysdiagnose completed. File path: /foo/bar.tar.gz', True),
         ('sysdiagnose', False),
         ('', False),
     )
 )
-def test_sysdiagnose_syslog_message_match_return_value(message, expected):
-    assert CrashReportsManager._sysdiagnose_complete_syslog_match(message) is expected
+def test_sysdiagnose_syslog_message_match_return_value(message, return_value):
+    assert CrashReportsManager._sysdiagnose_complete_syslog_match(message) is return_value
+
+
+@pytest.mark.parametrize(
+    ('end_time', 'return_value'),
+    ((-1, True), (0, True), (10, False), (None, False))
```

Comment:
this test fails

Reply:
Hey @doronz88 , is there a way that I can contribute to this PR?

Reply:
Feel free to fix this test yourself

Reply:
Ok, but that will create a new PR, won't it? I was thinking on this PR, which I cannot edit (or it seems so)